GameServer/Logic/Logger.cs

```diff
@@ -114,8 +114,8 @@ public void CreateLogFile(string directory, string name)
 
                 var logName = string.Format(
                     "{0}-{1}-{2}",
-                    DateTime.Now.ToShortDateString().Replace(".", ""),
-                    DateTime.Now.ToShortTimeString().Replace(" ", "").Replace(":", ""),
+                    DateTime.Now.ToShortDateString().Replace("/", "_"),
```

Comment:
The filename should be a string like
`20160921-1700-LeagueSandbox.txt`
so the format should be
`YYYYMMDD-HHMM-Name`

If you're changing this, make it follow that format please.
Not to mention this isn't a real fix. I'm willing to bet `ToShortDateString()` is locale dependent, and will give a different output depending on your locale. Use proper date/time formatting intead
https://msdn.microsoft.com/en-us/library/8kb3ddd4(v=vs.110).aspx